src/librustc_ast_passes/ast_validation.rs

```diff
@@ -1153,9 +1162,13 @@ impl<'a> Visitor<'a> for AstValidator<'a> {
         }) = fk.header()
         {
             self.err_handler()
-                .struct_span_err(span, "functions cannot be both `const` and `async`")
+                .struct_span_err(
+                    vec![*cspan, *aspan],
+                    "functions cannot be both `const` and `async`",
+                )
                 .span_label(*cspan, "`const` because of this")
                 .span_label(*aspan, "`async` because of this")
+                .span_label(span, "")
```

Comment:
What's up with this?

Reply:
The error is that there's both `const` and `async`, that's what should be underlined red. `span` contains the `def_span`, didn't want to completely remove it.

Reply:
I see. Leave a comment on the method call?